src/Makefile

```diff
@@ -27,6 +27,7 @@ NSS_CACHE_OSLOGIN_SONAME = libnss_cache_oslogin.so.2
 NSS_OSLOGIN              = libnss_oslogin-$(VERSION).so
 NSS_CACHE_OSLOGIN        = libnss_cache_oslogin-$(VERSION).so
 
+PAM_BYOID                = pam_oslogin_byoid.so
```

Comment:
Consider naming it as PAM Cert instead of PAM BYOID, the implementation is CERT specific, BYOID is just an use case.  We can envision that it will work for 1p account in the future

Reply:
Nope, this module is only intercepting the pam "auth" stack. The admin use case is being handled by "account" stack and nothing changes there.